pymc3/step_methods/hmc/nuts.py

```diff
@@ -12,6 +12,7 @@
 
 
 def bern(p):
+    assert np.isfinite(p)
```

Comment:
what are you checking for here? generally we should avoid assertions in favor of explicitly raising an exception.